spec/unit/installer/pod_source_installer_spec.rb

```diff
@@ -30,6 +30,22 @@ module Pod
           pod_folder = config.sandbox.pod_dir('BananaLib')
           pod_folder.should.exist
         end
+
+        it 'works fine if the source is encrypted using https' do
```

Comment:
nit: `works fine` is a bit generic here and can be `does not show warning ...`

Reply:
Done